CountOnMeTests/CalculatorModelTests.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  CalculatorManagerTests.swift
+//  CountOnMeTests
+//
+//  Created by Redouane on 07/03/2024.
+//  Copyright © 2024 Vincent Saluzzo. All rights reserved.
+//
+
+@testable import CountOnMe
+import XCTest
+
+typealias Operations =  CalculatorModel.MathOperator
+
+final class CalculatorManagerTests: XCTestCase {
+    var calculatorModel: CalculatorModel!
+    let numberFormatter: NumberFormatter = .init()
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        self.calculatorModel = .init()
+        numberFormatter.locale = .current
+        numberFormatter.decimalSeparator = "."
+        numberFormatter.minimumFractionDigits = 4
```

Comment:
**suggestion (non-blocking)**: Not needed to set those each time a test is set up.